Sources/BottomSheet/BottomSheet.swift

```diff
@@ -7,55 +7,84 @@
 
 import SwiftUI
 
-public struct BottomSheet<SheetContent:View, V: View>: View {
-    @Binding private var isPresented: Bool
-    @ViewBuilder private let sheetContent: () -> SheetContent
-    
-    private let view: V
-    
-    internal var configuration: BottomSheetViewConfiguration = .init()
-    var edges: Edge.Set?
-    
-    public init(isPresented: Binding<Bool>, sheetContent: @escaping () -> SheetContent, view: V) {
-        self._isPresented = isPresented
-        self.sheetContent = sheetContent
-        self.view = view
+public struct BottomSheet<Content: View, SheetContent: View>: View {
+    @State var configuration = BottomSheetViewConfiguration()
+
+    private let isPresented: Bool
+    private let content: Content
+    private let sheetContent: SheetContent
+
+    public init(
+        isPresented: Bool = true,
+        @ViewBuilder content: () -> Content,
+        @ViewBuilder sheetContent: () -> SheetContent
+    ) {
+        self.isPresented = isPresented
+        self.content = content()
+        self.sheetContent = sheetContent()
     }
     
     public var body: some View {
         ZStack {
-            view
+            content
             
             if isPresented {
-                BottomSheetView(configuration: configuration, content: sheetContent)
-                    .ignoresSafeAreaEdges(configuration.ignoredEdges)
```

Comment:
Hmmm I am realizing now that this isn't publicly exposed, I'll re-add

Reply:
In [434b17c](https://github.com/wojtek717/bottom-sheet/pull/8/commits/434b17c29607879805070c73ed1666b4363c9a56) I re-added support for ignoring safe area for the bottom sheet. 

In my opinion, `.bottom` should _not_ be the default value. This results `List` content being hidden behind the `TabView` which is typically not behavior people would like. Here are demo videos.

With `.bottom`:

https://github.com/user-attachments/assets/3cbf88dc-ee2c-4857-a740-1c96221e33d8

Without `.bottom`:

https://github.com/user-attachments/assets/2551a1e6-a357-4bdf-85cf-ea8d9c3ff7e4